lib/ansible/config/base.yml

```diff
@@ -930,6 +930,15 @@ DEFAULT_PRIVATE_KEY_FILE:
   ini:
   - {key: private_key_file, section: defaults}
   type: path
+DEFAULT_PRIVATE_KEY_FILE_PASSPHRASE:
+  name: Private key file passphrase
+  default: ~
+  description:
+    - Private key file passphrase
+  env: [{name: ANSIBLE_PRIVATE_KEY_FILE_PASSPHRASE}]
+  ini:
+  - {key: private_key_file_passphrase, section: defaults}
+  type: string
```

Comment:
this should not be a global config if it only applies to a specific connection plugin